.github/workflows/workflows.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: ai-cfia workflows
+
+on:
+  pull_request:
+    types:
+      - opened
+      - closed
+      - synchronize
+
+
+jobs:
+  python-lint:
+    name: workflow-lint-test-python
+    uses: ai-cfia/github-workflows/.github/workflows/workflow-lint-test-python.yml@main
+    secrets: inherit
+  mkd-check:
+    name: workflow-markdown-check
+    uses: ai-cfia/github-workflows/.github/workflows/workflow-markdown-check.yml@main
+    with:  # to pass inputs to called workflow
+      config-file-path: '.mlc_config.json'
```

Comment:
This is  supposed to be provided. Its not in your added files. You can either add this file with custom rules or remove these lines to use default rules.

Reply:
Fixed